plugins/ssr/ssrparser.mlg

```diff
@@ -101,14 +106,15 @@ let register_ssrtac name f prods =
     | TacNonTerm (_, (_, ido)) -> ido in
   let ids = List.map_filter get_id prods in
   let tac = TacML (CAst.make (ssrtac_entry name, List.map map ids)) in
-  let key = Lib.make_kn (Names.Id.of_string_soft ("ssreflect_plugin@#" ^ name)) in
+  let key = Lib.make_kn (Names.Id.of_string_soft (__coq_plugin_name ^ "_" ^ name)) in
```

Comment:
You can't use Lib.make_kn in this context. You have to generate the kername yourself.

Reply:
Okay, how about this?

Reply:
Looks way better to me now.